### ISSUE TITLE
ERROR: The specified art folder (/tmp/libbieoffice_mod//20170826_tysontan_libbie) doesn't exist!

### ISSUE THREAD

**Issue by brlin-tw** [calebj/LibbieOffice#1]
I noticed two issues about this message:

1. The log level shouldn't be ERROR, as it is normal that the art folder doesn't exist without running the mod setup script once, maybe INFO would be proper
1. The art folder path contains redundant slashes

The script does seems to be effective as the splash screen mod is applied successfully.

```
$ sudo ./libbieoffice_mod.sh
ERROR: The specified art folder (/tmp/libbieoffice_mod//20170826_tysontan_libbie) doesn't exist!
Downloading art zip from dropbox...
/tmp/libbieoffice_mod//2017 100%[===========================================>]  37.93M  3.32MB/s    in 14s     
2017-11-21 07:15:04 URL:https://dl.dropboxusercontent.com/content_link/goOWukmk4cGm5WnKhMsOkBfzzQSBx8iBhI3MrlgHQ7Hsi7upaDX3lrv87xTHmZwe/file?dl=1 [39773852/39773852] -> "/tmp/libbieoffice_mod//20170826_tysontan_libbie_full.zip" [1]
Extracting...
Generating color icons:
 - base... ok.
 - calc... ok.
 - draw... ok.
 - impress... ok.
 - math... ok.
 - writer... ok.
 - blank... ok.
Generating blackwhite icons:
 - base... ok.
 - calc... ok.
 - draw... ok.
 - impress... ok.
 - math... ok.
 - writer... ok.
 - blank... ok.
Building splash from components libbie + 34view
Installing color icons...
 - gnome... ok.
 - hicolor... ok.
Installing blackwhite icons...
 - HighContrast... ok.
Modifying color theme zips...
 - galaxy... ok.
 - breeze... not found.
 - breeze_dark... not found.
 - tango... ok.
Modifying blackwhite theme zips...
 - hicontrast... not found.
```

**Comment by calebj** [comment]
Yeah, both are issues are just cosmetic.

**Comment by calebj** [comment]
Fixed in baadc4ed54f692d8a75bcfdcb7b8ae9566c63942, spamming another comment because I am a dumb and forgot to put `resolves #1` in the commit message.
